api/openapi.yaml

```diff
@@ -1237,6 +1237,11 @@ components:
           $ref: '#/components/schemas/ScanFamiliesConfig'
         scope:
           $ref: '#/components/schemas/ScanScopeType'
+        timeoutSeconds:
```

Comment:
need to update the odata schema for the api changes

Reply:
Nice catch, keep forgetting to do that. Will fix it now.

Reply:
Fixed.